<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>119</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="chatButton">
      <property name="text">
       <string>Chat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fileButton">
      <property name="text">
       <string>Archivos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>117</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Desconectarse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
